types/index.d.ts

```diff
@@ -46,21 +53,21 @@ export class AedesMemoryPersistence {
 
   addSubscriptions: (
     client: Client,
-    subs: AedesPersistenceSubscription[],
+    subs: Subscription[],
     cb: (error: CallbackError, client: Client) => void
   ) => void;
 
   removeSubscriptions: (
     client: Client,
-    subs: AedesPersistenceSubscription[],
+    subs: Subscription[],
     cb: (error: CallbackError, client: Client) => void
   ) => void;
 
   subscriptionsByClient: (
     client: Client,
     cb: (
       error: CallbackError,
-      subs: AedesPersistenceSubscription[],
+      subs: { topic: string; qos: QoS }[],
```

Comment:
Isn't this `Subscription[]`?

Reply:
This is happening in `subscriptionsByClient` :
```js
    ...
    var subs = []
    subs.push({ topic: topicAndQos[0], qos: topicAndQos[1] })
    ...
```